mautic/mautic
# Multiple form on one page validation fix

Reported here https://www.mautic.org/community/index.php/556-multiple-forms-on-one-page

When 2 different forms are embedded to one page and both has field with the same name ("email" for example) the validation messages shows up only on the first form even though the second one was submitted.

#### How to test
1. Create the first form with required email field.
2. Create the second form with required email field.
3. Create a Landing Page.
4. Embed both forms to the Landing page.
5. Open the landing page on public URL
6. Try to submit the second form with empty email field.

Result: The error message shows up at the email field of the first form.

After applying this PR the message should appear on the form where the submit button was pressed.

Please, test also other fields than just email. The code had to be changed for all of them.

This PR also fixes populating field values from URL. For example:
```http://your-mautic.com/form/1/?email=leademail%40gmail.com``` wouldn't prefill the email field before the PR, but it does prefill it after this PR.
